packages/ember-glimmer/tests/integration/helpers/closure-action-test.js

```diff
@@ -689,10 +689,10 @@ moduleFor('Helpers test: closure {{action}}', class extends RenderingTest {
     });
 
     let OuterComponent = Component.extend({
-      init() {
-        this._super(...arguments);
-        outerComponent = this;
-      },
+      // init() {
```

Comment:
If the tests pass without it, just delete it.  We can always get it back if needed 😸